docker-compose.override.yml.template.multievent

```diff
@@ -31,6 +32,7 @@ x-dev-volumes:
     - &plugin-config-file-1 MAGSTOCK_CONFIG_FILES=config.ini
     - &plugin-config-file-2 MAGWEST_CONFIG_FILES=config.ini
     - &plugin-config-file-3 MAGPRIME_CONFIG_FILES=config.ini
```

Comment:
Is there a reason to rename everything to config.ini rather than use:
```suggestion
    - &plugin-config-file-1 MAGSTOCK_CONFIG_FILES=magstock.ini
    - &plugin-config-file-2 MAGWEST_CONFIG_FILES=magwest.ini
    - &plugin-config-file-3 MAGPRIME_CONFIG_FILES=magprime.ini
```

Reply:
Hm, good point. That's to mirror the one-event setup, which uses config.ini as the filename so you only have to edit the name of the _CONFIG_FILES env var. We can take it out of this setup though.